sample-windows.ini

```diff
@@ -14,9 +14,11 @@ portNumber = 8000
 remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
 ; These options are unlikely to need editing by users
-maxCacheEntries = 2000000
-mruSymbolStateFile = %(TEMP)s\snappy-mru-symbols.json
-maxMRUSymbolsPersist = 10
+maxCacheEntries = 400
```

Comment:
I changed the code for when `maxMemCacheFiles` and `maxDiskCacheFiles` are zero, cache is disabled. This is the default on `sample-windows.ini`.